docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,93 @@
 # Adding a feed
 
  1. To add a feed, copy the URL of its RSS or Atom file (for instance, the Framablog RSS URL is `https://framablog.org/feed/`). FreshRSS is able to automatically find the address of the feed for websites that are declaring it in a standard way.
- 2. On FreshRSS, click “Subscriptions management”.
- 3. Paste the URL in “Add an RSS feed” below the page title.
+ 2. On FreshRSS, click the "**+**" button next to “Subscriptions management”.
+ 3. Paste the URL in “Feed URL” field.
  4. (optional): You can select the category for your feed. By default, it will be in “Uncategorized”.
+ 5. (optional): If the subscription requires credentials, you can enter them in the "HTTP username" and "HTTP password" fields. 
+ 6. (optional): You can get a timeout for the feed request if the feed requires it.
+ 7. (optional): You can choose to ignore SSL certificate errors (such as with self-signed certificates) by setting "Verify SSL security" to "No". This is not recommended, and it is better to either add the root certificate to the FreshRSS server or to fix the SSL certificate problems on the feed hosting server. 
 
-# Import and export
+# Subscription management
+
+The "Subscription management" submenu allows categories and feeds to be configured. Feeds can be moved between categories by drag-and-drop, or in the individual feed's settings. Hovering over a feed/category will cause a gear icon to appear. Clicking the icon will bring up the settings for that item. 
+
+## Category Settings
+
+### Information 
+
+* **Title:** Name of category
+* **Display position:** Defines order of categories. Lower numbers get priority, non-numbered items come last, and equally numbered items will sort by alphabetical order.
+
+### Archiving
+
+If "Purge Policy" has "By default" selected, then the [default purge policy](./05_Configuration.md) is used and the other options are not displayed. Category options will override the default policy, but will not override feed-specific options.
+
+## Feed Settings
+
+### Information
+
+These should be auto-filled when adding a feed, but can be modified after. **Visibility** will define if the feed is displayed in the main feed, only in category feeds, or not at all.
+
```

Comment:
It's generally preferred not to use subheadings immediately after another (sub)heading), and to open with a text very much like this instead! ;-)
```suggestion

These fields will be auto-filled when adding a feed, but they can be modified later. **Visibility** will define if the feed is displayed in the main feed, only in specific categories, or not at all.

```